hardware/Layout - desing- guide/Layout-Desing-Guide.md

```diff
@@ -0,0 +1,464 @@
+
+# EFR32 Layout Design Guide
+
+The purpose of this material is to help users design PCBs
+for the EFR32 Wireless Gecko Portfolio using design practices
+that allow for good RF performance.
+
+PCB layout requirements are described in the application note, [AN928.1: EFR32 Series 1 Layout Design Guide](https://www.silabs.com/documents/public/application-notes/an928.1-efr32-series1-layout-design-guide.pdf),  and the layout design process for EFR32 Series 1 Modules is discussed in [the datasheet of each module](https://www.silabs.com/support/resources.ct-data-sheets.ct-miscellaneous.p-wireless_bluetooth-low-energy_blue-gecko-bluetooth-low-energy-modules.p-wireless_bluetooth-low-energy_blue-gecko-bluetooth-low-energy-modules_bgm11s12f256ga-v2.p-wireless_bluetooth-low-energy_efr32bg1-series-1-modules.p-wireless_bluetooth-low-energy_efr32bg13-series-1-modules.sort=2,asc) and  beside that EFR32 reference design files are available in Simplicity Studio.
+
+
+## Why is the proper layout design important? 
+
+If the layout is a inappropriate design that can cause 
++ Degradation in the communication distance 
++ The unit can violate the relevant EMC regulation (high harmonics, spur) 
++ The battery life can be reduced
+
+## RF section 
+
+
+### Different parts of RF section
+The most important part of the proper layout design is the RF circuitry itself.
+
+In the case of dual band RFIC, this RF part includes the following:
+1. Sub-GHz and 2.4 GHz RF matching newtork  
+2. Power supply filterings 
+3. High frequency crystal  
+
+
+
+<p align="center">
+  "<img src="matching network.png">
+</p>
+
+<center> 
```

Comment:
make the name of figure center alignment, the same applies all the figure name in this doc
<div align="center">
  <b>Figure 1. RF section</b>
</div>

Reply:
Your suggestion:

<div align="center">
  <b>Figure 1. RF section</b>
</div>
  
and my solution:

<center> 
**Figure 1. RF section** 
</center>

can do the same.

Are your sure I should change it?

Reply:
It is okay if that works. However, from the preview of the file. It looks it doesn't align with center